canvas_editor/canvas/message_dict.py

```diff
@@ -0,0 +1,69 @@
+# Account specifiy messages
+
+password_length_criterium_text = "Password must be at least 8 characters long."
```

Comment:
```suggestion
password_length_criterion_text = "Password must be at least 8 characters long."
```
criterion would be the correct translation of Kriterium → should be updated throughout